test/convert.jl

```diff
@@ -222,3 +222,18 @@ end
     x1 = pyconvert(DateTime, pydatetime(2001, 2, 3, 4, 5, 6, 7000))
     @test x1 === DateTime(2001, 2, 3, 4, 5, 6, 7)
 end
+
+@testitem "pyconvert_add_rule (#364)" begin
```

Comment:
Because of the stateful nature of the conversion rules, this test fails if you run it twice in succession in the same Julia session (e.g. from the VS Code testing pane). A fix is to use a random class name, such as `class = "Hello_364_$(string(rand(UInt), base=16))"`.